Update documentation to include experimental features (#2970)
modin-project/modin: docs/experimental_features/modin_sql.rst

    LIMIT 10
    """

    # Run query using apps and reviews dataframes, 
    # NOTE: that you simply pass the names of the tables in the query as arguments

    result_df = sql.query( sql_str, 
                            gstore_apps_df = gstore_apps_df, 
                            gstore_reviews_df = gstore_reviews_df)

Or, you can bring the best of doing this in python and run the query in multiple parts (it’s up to you). 

.. code-block:: python

    # join the items and reviews

    result_df = sql.query( """ 
    SELECT 
        category, 
        sentiment, 
        sentiment_polarity 
    FROM gstore_apps_df INNER JOIN gstore_reviews_df 
    ON gstore_apps_df.app = gstore_reviews_df.app """, 
    gstore_apps_df = gstore_apps_df, 
    gstore_reviews_df = gstore_reviews_df )

    # group by category and calculate averages

    result_df = sql.query( """
    SELECT 
        category, 
        avg(sentiment_polarity) as avg_sentiment_polarity, 
        avg(sentiment_subjectivity) as avg_sentiment_subjectivity 
    GROUP BY category
    HAVING CAST(avg_sentiment_subjectivity as float) < 0.5
    ORDER BY avg_sentiment_polarity DESC
    LIMIT 10""", 
    from = result_df)

> The crazy thing here is that if you have a cluster or even a computer with more than one core, 

Reviewer: ```suggestion
If you have a cluster or even a computer with more than one CPU core, 
```
